nomad/job_endpoint.go

```diff
@@ -34,6 +34,9 @@ const (
 	DispatchPayloadSizeLimit = 16 * 1024
 )
 
+// ErrMultipleNamespaces is send when multiple namespaces are used in the OSS setup
+var ErrMultipleNamespaces = errors.New("multiple vault namespaces requires Nomad Enterprise")
```

Comment:
we should capitalize Vault as well, but that's not worth worrying about now :sweat_smile: